lib/web/ui/webconfig.go

```diff
@@ -69,4 +69,8 @@ type WebConfigAuthSettings struct {
 	LocalAuthEnabled bool `json:"localAuthEnabled"`
 	// AuthType is the authentication type.
 	AuthType string `json:"authType"`
+	// PreferredLocalMFA is a server-side hint for clients to pick an MFA method
+	// when various options are available.
+	// It is empty if there is nothing to suggest.
+	PreferredLocalMFA constants.SecondFactorType `json:"preferred_local_mfa"`
```

Comment:
could we use camel case? it aligns with JS camelcase conventions. I know it seems like a mess that we use both :sweat:, but i think that was in the earlier days.

Reply:
On second though that looks weird, changed to `preferredLocalMFA`.

Reply:
you were right the first time `preferredLocalMfa`, there was a reason why we don't all cap the acronym but don't remember specifically @alex-kovoy

Reply:
@codingllama it's better to `preferredLocalMfa` as it's aligned with our naming convention. Proto to JS generators also stick with that too. So for the `preferredLocalMFA` golang field the `preferrenMfa` JS will be generated. At least by default.

Reply:
+1 for following the [Go naming conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) in Go source code. It's fine if the JS is slightly different IMO.

Reply:
No strong opinions about json tags on my part, changed to `preferredLocalMfa`.